source/base/Text.ooc

```diff
@@ -164,6 +169,16 @@ Text: cover {
 		separator free(Owner Receiver)
 		result
 	}
+	trim: func -> This {
+		length := this count
```

Comment:
It reduces readability in favor for a potential performance improvement that is not needed. It adds one line of code and by that makes maintenance harder.